content/tutorials/single-page-tutorials/send-shimmer-tokens-with-javascript.md

```diff
@@ -0,0 +1,276 @@
+# Send Shimmer Tokens with Javascript (client lib)
```

Comment:
I think we can remove `client lib`from the title as it is already clear that is is by the tag 🤔  

```suggestion
# Send Shimmer Tokens with Javascript
```

Reply:
Removed